Deprecation throws ValueError

While updating the package in Fedora to 0.13.4, I was testing the package dependencies and see that ephyviewer has a failing test now:

```
=================================== FAILURES ===================================
_______________________ test_traceviewer_cls_method_neo ________________________

interactive = False

    def test_traceviewer_cls_method_neo(interactive=False):
        sigs = np.random.rand(100000,16)
        sample_rate = 1000.
        t_start = 0.
    
>       neo_anasig = neo.AnalogSignal(sigs*pq.mV, sampling_rate=sample_rate*pq.Hz, t_start=0*pq.s, copy=True)

ephyviewer/tests/test_traceviewer.py:74: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

cls = <class 'neo.core.analogsignal.AnalogSignal'>
signal = array([[0.42289109, 0.87757422, 0.97128633, ..., 0.42063227, 0.33470735,
        0.57933975],
       [0.39828568, 0.78...       0.87677851],
       [0.38279604, 0.85713246, 0.43967717, ..., 0.69529369, 0.73503751,
        0.0068759 ]]) * mV
units = None, dtype = None, copy = True, t_start = array(0.) * s
sampling_rate = array(1000.) * Hz, sampling_period = None, name = None
file_origin = None, description = None, array_annotations = None
annotations = {}

    def __new__(
        cls,
        signal,
        units=None,
        dtype=None,
        copy=None,
        t_start=0 * pq.s,
        sampling_rate=None,
        sampling_period=None,
        name=None,
        file_origin=None,
        description=None,
        array_annotations=None,
        **annotations,
    ):
        """
        Constructs new :class:`AnalogSignal` from data.
    
        This is called whenever a new class:`AnalogSignal` is created from
        the constructor, but not when slicing.
    
        __array_finalize__ is called on the new object.
        """
        if copy is not None:
>           raise ValueError(
                "`copy` is now deprecated in Neo due to removal in NumPy 2.0 and will be removed in 0.15.0."
            )
E           ValueError: `copy` is now deprecated in Neo due to removal in NumPy 2.0 and will be removed in 0.15.0.

/usr/lib/python3.13/site-packages/neo/core/analogsignal.py:202: ValueError
```

Is the intention here to throw an error (that makes the code stop)? Or is it to inform the user that they need to update their code---in which case, the deprecation warning should not make code fail (using a DeprecationWarning perhaps: https://docs.python.org/3/library/exceptions.html#DeprecationWarning ?)

--- a/neo/core/analogsignal.py
+++ b/neo/core/analogsignal.py
@@ -200,7 +200,10 @@ class AnalogSignal(BaseSignal):
         """
         if copy is not None:
             raise ValueError(
-                "`copy` is now deprecated in Neo due to removal in NumPy 2.0 and will be removed in 0.15.0."
+                "`copy` is now deprecated in Neo due to removal in Quantites to support Numpy 2.0. "
+                "In order to facilitate the deprecation copy can be set to None but will raise an "
+                "error if set to True/False since this will silently do nothing. This argument will be completely "
+                "removed in Neo 0.15.0. Please update your code base as necessary."
             )
 
         signal = cls._rescale(signal, units=units)
@@ -210,7 +213,7 @@ class AnalogSignal(BaseSignal):
             obj.shape = (-1, 1)
 
         if t_start is None:
-            raise ValueError("t_start cannot be None")
+            raise ValueError("`t_start` cannot be None")
         obj._t_start = t_start
 
         obj._sampling_rate = _get_sampling_rate(sampling_rate, sampling_period)

--- a/neo/core/imagesequence.py
+++ b/neo/core/imagesequence.py
@@ -124,7 +124,10 @@ class ImageSequence(BaseSignal):
 
         if copy is not None:
             raise ValueError(
-                "`copy` is now deprecated in Neo due to removal in NumPy 2.0 and will be removed in 0.15.0."
+                "`copy` is now deprecated in Neo due to removal in Quantites to support Numpy 2.0. "
+                "In order to facilitate the deprecation copy can be set to None but will raise an "
+                "error if set to True/False since this will silently do nothing. This argument will be completely "
+                "removed in Neo 0.15.0. Please update your code base as necessary."
             )
 
         if spatial_scale is None:

--- a/neo/core/irregularlysampledsignal.py
+++ b/neo/core/irregularlysampledsignal.py
@@ -174,7 +174,10 @@ class IrregularlySampledSignal(BaseSignal):
 
         if copy is not None:
             raise ValueError(
-                "`copy` is now deprecated in Neo due to removal in NumPy 2.0 and will be removed in 0.15.0."
+                "`copy` is now deprecated in Neo due to removal in Quantites to support Numpy 2.0. "
+                "In order to facilitate the deprecation copy can be set to None but will raise an "
+                "error if set to True/False since this will silently do nothing. This argument will be completely "
+                "removed in Neo 0.15.0. Please update your code base as necessary."
             )
 
         signal = cls._rescale(signal, units=units)

--- a/neo/core/spiketrain.py
+++ b/neo/core/spiketrain.py
@@ -194,7 +194,12 @@ def normalize_times_array(times, units=None, dtype=None, copy=None):
     """
 
     if copy is not None:
-        raise ValueError("`copy` is now deprecated in Neo due to removal in NumPy 2.0 and will be removed in 0.15.0.")
+        raise ValueError(
+            "`copy` is now deprecated in Neo due to removal in Quantites to support Numpy 2.0. "
+            "In order to facilitate the deprecation copy can be set to None but will raise an "
+            "error if set to True/False since this will silently do nothing. This argument will be completely "
+            "removed in Neo 0.15.0. Please update your code base as necessary."
+            )
 
     if dtype is None:
         if not hasattr(times, "dtype"):
@@ -352,7 +357,10 @@ class SpikeTrain(DataObject):
         """
         if copy is not None:
             raise ValueError(
-                "`copy` is now deprecated in Neo due to removal in NumPy 2.0 and will be removed in 0.15.0."
+                "`copy` is now deprecated in Neo due to removal in Quantites to support Numpy 2.0. "
+                "In order to facilitate the deprecation copy can be set to None but will raise an "
+                "error if set to True/False since this will silently do nothing. This argument will be completely "
+                "removed in Neo 0.15.0. Please update your code base as necessary."
             )
 
         if len(times) != 0 and waveforms is not None and len(times) != waveforms.shape[0]:

